assets/alice.gui

```diff
@@ -110,73 +110,37 @@ guiTypes = {
 			}
 
 			instantTextBoxType={
-				position = {x = 58 y = 104 }
+				position = {x = 68 y = 115 }
 				name = "window_mode_label"
-				font = "Arial16"
-				borderSize = {x = 0 y = 0}
-				maxWidth = 236
-				maxHeight = 18
-				text = "window_mode"
-				orientation = "UPPER_LEFT"
-				format = centre
-			}
-
-			instantTextBoxType={
-				position = {x = 58 y = 122 }
-				name = "window_mode_value"
+				text = "window_mode_label"
 				font = "Arial16"
 				borderSize = {x = 0 y = 0}
 				maxWidth = 236
 				maxHeight = 18
 				orientation = "UPPER_LEFT"
 				format = centre
 			}
-
 			guiButtonType = {
-				name = "window_mode_left"
+				name = "window_mode_checkbox"
 				position = { x = 51 y = 115 }
-				quadTextureSprite = "option_prev"
-			}
-
-			guiButtonType = {
-				name = "window_mode_right"
-				position = { x = 269 y = 115 }
-				quadTextureSprite = "option_next"
+				quadTextureSprite = "GFX_checkbox_default"
 			}
 
 			instantTextBoxType={
-				position = {x = 58 y = 157 }
+				position = {x = 68 y = 168 }
 				name = "projection_label"
+				text = "map_projection_label"
 				font = "Arial16"
 				borderSize = {x = 0 y = 0}
 				maxWidth = 236
 				maxHeight = 18
-				text = "map_projection"
 				orientation = "UPPER_LEFT"
-				format = centre
-			}
-
-			instantTextBoxType={
-				position = {x = 58 y = 175 }
-				name = "projection_value"
-				font = "Arial16"
-				borderSize = {x = 0 y = 0}
-				maxWidth = 236
-				maxHeight = 18
-				orientation = "UPPER_LEFT"
-				format = centre
+				format = left
 			}
-
 			guiButtonType = {
-				name = "projection_left"
+				name = "projection_checkbox"
```

Comment:
you can't reduce the projections to a checkbox because I probably eventually want to add more of them